batch_carolapp/spec.json

```diff
@@ -5,13 +5,13 @@
   "memory": 4000,
   "source": {
     "type": "git",
-    "path": "https://${GHTOKEN}@github.com/totvslabs/carolapp-samples",
+    "path": "https://${GHTOKEN}@github.com/totvslabs/batch_carolapp",
```

Comment:
this file is not needed.. This is to build directusing operator's APIS no?

Reply:
Ok, removing the file here and in cookie-cutter project.